packages/nimble/inst/CppCode/eigenUsingClasses.cpp

```diff
@@ -7,44 +7,46 @@
 
```

Comment:
WDYT about moving these template specializations to their neighbors in `RcppNimbleUtils.h`, in a follow-up PR?